src/server/destination/destination-type/index.js

```diff
@@ -38,14 +36,7 @@ export class DestinationTypePage extends QuestionPage {
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   nextPage(answer, context) {
-    if (
-      !config.get('featureFlags').animalIdentifiers &&
-      context.origin?.onOffFarm === 'on' &&
-      restricted.includes(context.origin?.originType) &&
-      possibleRestricted.includes(answer.value)
-    ) {
-      return destinationNotSupportedPage
-    } else if (context.origin?.onOffFarm === 'on') {
+    if (context.origin?.onOffFarm === 'on') {
```

Comment:
Do we use the 'possibleRestricted' and 'restricted' arrays any more?  I think the can be removed / moved to the test file at least.